python/tvm/autotvm/graph_tuner/base_graph_tuner.py

```diff
@@ -0,0 +1,551 @@
+# pylint: disable=too-many-arguments,too-many-locals,too-many-statements,too-many-instance-attributes,too-many-branches,too-many-nested-blocks,invalid-name,unused-argument,unused-variable,no-member
+"""Base class for graph tuner."""
+import logging
+from abc import abstractmethod
+
+import topi
+
+import tvm
+from tvm import autotvm, relay
+from tvm.autotvm.task import get_config
+from tvm.autotvm.task.topi_integration import deserialize_args, serialize_args
+from tvm.autotvm.record import encode, load_from_file
+from tvm.autotvm.measure import MeasureResult, MeasureInput
+from tvm.relay.expr import Call, Var, Constant
+
+from .utils import is_input_node, shape2layout, get_in_nodes, get_out_nodes, \
+    has_multiple_inputs, get_wkl_map, bind_inputs, expr2graph
+
+
+@autotvm.template
+def layout_transform(*args):
+    """Autotvm layout transform template."""
+    args = deserialize_args(args)
+    cfg = get_config()
+    cfg.add_flop(-1)
+    data = args[0]
+    out = topi.nn.layout_transform(*args)
```

Comment:
There is no ```topi.nn.layout_transform```. We have ```topi.layout_transform``` instead, [here](https://github.com/dmlc/tvm/blob/master/topi/python/topi/transform.py#L381).

But I still get the following error. Maybe the API changed recently?
```
TypeError: layout_transform() takes 3 positional arguments but 6 were given
```

Reply:
yes top.nn.layout_transform was changed recently.